strictdoc/export/html2pdf/html2pdf.py

```diff
@@ -211,14 +213,19 @@ def main():
     os.environ["WDM_LOCAL"] = "1"
 
     parser = argparse.ArgumentParser(description="HTML2PDF printer script.")
+    parser.add_argument(
```

Comment:
Let's document this shortly in the User Guide. Also mention that the Chrome Driver / Browser are used as a printing machine.

Reply:
How should we support this for `strictdoc server`? The web UI can initiate the pdf export as well, so it would need either the same CLI argument, or it would have to be parsed from strictdoc.toml.

Reply:
Ideally, it should support both CLI and TOML. But as with everything else, this can be postponed unless you need this right away. Do whatever you have resources for.